contracts/Core/storage/BlockStorage.sol

```diff
@@ -18,4 +18,6 @@ contract BlockStorage {
     int8 public blockIndexToBeConfirmed; // Index in sortedProposedBlockIds
     // epoch -> blocks
     mapping(uint32 => Structs.Block) public blocks;
+
+    bytes32 public salt;
```

Comment:
This is already present in voteManager so don't need it here.

Reply:
yep, good catch 
lets remove it

Reply:
resolved